src/Endpoints/Snapshot.php

```diff
@@ -279,25 +285,25 @@ public function deleteRepository(array $params = [])
-	 *     master_timeout: time, // Explicit operation timeout for connection to master node
-	 *     ignore_unavailable: boolean, // Whether to ignore unavailable snapshots, defaults to false which means a SnapshotMissingException is thrown
-	 *     index_names: boolean, // Whether to include the name of each index in the snapshot. Defaults to true.
-	 *     index_details: boolean, // Whether to include details of each index in the snapshot, if those details are available. Defaults to false.
-	 *     include_repository: boolean, // Whether to include the repository name in the snapshot info. Defaults to true.
-	 *     sort: enum, // Allows setting a sort order for the result. Defaults to start_time
-	 *     size: integer, // Maximum number of snapshots to return. Defaults to 0 which means return all that match without limit.
-	 *     order: enum, // Sort order
-	 *     from_sort_value: string, // Value of the current sort column at which to start retrieval.
-	 *     after: string, // Offset identifier to start pagination from as returned by the 'next' field in the response body.
-	 *     offset: integer, // Numeric offset to start pagination based on the snapshots matching the request. Defaults to 0
-	 *     slm_policy_filter: string, // Filter snapshots by a comma-separated list of SLM policy names that snapshots belong to. Accepts wildcards. Use the special pattern '_none' to match snapshots without an SLM policy
-	 *     verbose: boolean, // Whether to show verbose snapshot info or only show the basic info found in the repository index blob
-	 *     pretty: boolean, // Pretty format the returned JSON response. (DEFAULT: false)
-	 *     human: boolean, // Return human readable values for statistics. (DEFAULT: true)
-	 *     error_trace: boolean, // Include the stack trace of returned errors. (DEFAULT: false)
-	 *     source: string, // The URL-encoded request definition. Useful for libraries that do not accept a request body for non-POST requests.
-	 *     filter_path: list, // A comma-separated list of filters used to reduce the response.
+	 *     snapshot: string, // (REQUIRED) A comma-separated list of snapshot names
+	 *     master_timeout?: int|string, // Explicit operation timeout for connection to master node
+	 *     ignore_unavailable?: bool, // Whether to ignore unavailable snapshots, defaults to false which means a SnapshotMissingException is thrown
+	 *     index_names?: bool, // Whether to include the name of each index in the snapshot. Defaults to true.
+	 *     index_details?: bool, // Whether to include details of each index in the snapshot, if those details are available. Defaults to false.
+	 *     include_repository?: bool, // Whether to include the repository name in the snapshot info. Defaults to true.
+	 *     sort?: string, // Allows setting a sort order for the result. Defaults to start_time
+	 *     size?: integer, // Maximum number of snapshots to return. Defaults to 0 which means return all that match without limit.
```

Comment:
Since you changed `boolean` to `bool` you might also want to change `integer` to `int`